customer-config/checkout-config/che.business-key-value.v1.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/extenda/hiiretail-json-schema-registry/master/customer-config/checkout-config/che.business-key-value.v1.json",
+    "title": "Business key-value configuration",
+    "type": "array",
+    "items": {
```

Comment:
Does it make sense to store object at top level? You have unique keys anyway in your array, might as well just have an object with keys...

That might simplify "patching" of config later on (new feature that we are working on)